shared/middleware/net_sim_api.rb

```diff
@@ -420,7 +393,7 @@ def redis_group_for_shard(shard_id)
   #
   # @return [Hash<'master':String, 'read_replicas':String[]>[]]
   def redis_groups
-    CDO.netsim_redis_groups || [{'master': 'redis://localhost:6379'}]
+    CDO.netsim_redis_groups || [{'master' => 'redis://localhost:6379'}]
```

Comment:
I messed up Ruby syntax and this was working accidentally.  I need the key to be a string because that's the format we get back from our configuration.  Writing this as `{'master': '...'}` makes the key a symbol.  Everything still worked because when we eventually asked for `config['master']` and got `nil` that's what we passed to the Redis constructor, which in turn defaulted to localhost and port 6379.

Reply:
Should this url be in a constant?

Reply:
Done.